src/classlibnative/bcltype/stringnative.h

```diff
@@ -85,10 +85,8 @@ class COMString {
     static FCDECL2(VOID,        FCSetTrailByte,    StringObject* thisRefUNSAFE, UINT8 bData);
 #endif // FEATURE_COMINTEROP
 
-#ifdef FEATURE_RANDOMIZED_STRING_HASHING
     static FCDECL3(INT32, Marvin32HashString, StringObject* thisRefUNSAFE, INT32 strLen, INT64 additionalEntropy);
     static BOOL QCALLTYPE UseRandomizedHashing();
```

Comment:
You should be able to remove all code that checks whether the randomized string hashing is enabled. It is effectively dead code - there is no supported way to turn it off in .NET Core.